comments-bundle/src/ContaoManager/Plugin.php

```diff
@@ -27,7 +27,7 @@ public function getBundles(ParserInterface $parser): array
     {
         return [
             BundleConfig::create(ContaoCommentsBundle::class)
-                ->setLoadAfter([ContaoCoreBundle::class])
+                ->setLoadAfter([ContaoCoreBundle::class, 'calendar', 'faq', 'news'])
```

Comment:
This should use the bundle names, not the legacy module names.

Reply:
@aschempp using the bundle names would need use statements which would ultimately lead to have the bundles installed again 🤔?

Reply:
no, use statements are not package requirements. A class name is just a string internally, the class does not have to be present.